packages/api/tsconfig.json

```diff
@@ -23,7 +23,8 @@
 
     "sourceMap": false /* Create source map files for emitted JavaScript files. */,
     "rootDir": "./src" /* Specify the root folder within your source files. */,
-    "outDir": "./dist" /* Specify an output folder for all emitted files. */
+    "outDir": "./dist" /* Specify an output folder for all emitted files. */,
+    "noErrorTruncation": false,
```

Comment:
OOC - why do we want the root tsconfig to be `true`, and this one `false`?

Reply:
good catch, this should be true - I likely changed while testing and forgot to revert